l1-contracts/src/core/libraries/ConstantsGen.sol

```diff
@@ -44,6 +45,7 @@ library Constants {
   uint256 internal constant MAX_NEW_L2_TO_L1_MSGS_PER_TX = 2;
   uint256 internal constant MAX_NEW_CONTRACTS_PER_TX = 1;
   uint256 internal constant MAX_READ_REQUESTS_PER_TX = 128;
+  uint256 internal constant MAX_NULLIFIER_READ_REQUESTS_PER_TX = 8;
```

Comment:
Can't compile if this number is bigger. We can set it to a larger number once it's done in a reset circuit.